mastracode/factory-ui/src/ui/domains/workspaces/components/sessionActivity.css

```
@@ -0,0 +1,333 @@
+ * without snapping at the loop seam, so `--wick-fill` brings the rim up under
+ * it. */
+@property --wick-run {
+  syntax: '<angle>';
+  inherits: false;
+  initial-value: 0deg;
+}
+
+.session-wick {
+  --belt-beat: 1.4s;
+  /* Lap length, in belt beats. Long, because the head is crossing a whole card
+   * rather than a 6px rail: it has to read as travelling, not as spinning. */
```

Stylelint (17.14.0)

[error] Expected empty line before comment (comment-empty-line-before)

(comment-empty-line-before)